src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -16,7 +16,10 @@ public enum JsonCommentHandling : byte
     public sealed partial class JsonDocument : System.IDisposable
     {
         internal JsonDocument() { }
+        public bool IsDetached { get { throw null; } }
+        public bool IsDisposable { get { throw null; } }
         public System.Text.Json.JsonElement RootElement { get { throw null; } }
+        public System.Text.Json.JsonDocument Detach(bool useArrayPools = false) { throw null; }
```

Comment:
Why plural? `useArrayPool`?

Reply:
> Why plural? `useArrayPool`?

I could go either way, really.  Since we talked about moving the database from byte to uint there are possibly technically two different pools involved in the future.

Reply:
It's minor, but I'd prefer something instead like `bool poolArrays`.